src/os/bluestore/BlueRocksEnv.cc

```diff
@@ -440,7 +440,13 @@ rocksdb::Status BlueRocksEnv::GetChildren(
   std::vector<std::string>* result)
 {
   result->clear();
-  int r = fs->readdir(dir, result);
+  // dir may contain a trailing /
```

Comment:
wouldn't it be better to put this inside readdir?

Reply:
indeed! pushed it down to `BlueFS::readdir()`.